src/components/ThemedImage/index.tsx

```diff
@@ -30,7 +30,9 @@ export function ThemedImage(props: ThemedImageProps) {
       {renderedSourceNames.map((sourceName) => (
         <img
           key={sourceName}
-          src={sources![sourceName as keyof Record<'light' | 'dark', string>]}
+          src={
+            sources !== undefined ? sources[sourceName as 'light' | 'dark'] : ''
+          }
```

Comment:
Instead of passing empty string as `img` attribute I'd rather use conditional rendering:

```jsx
  {renderedSourceNames.map((sourceName) =>
        sources ? (
          <img
            key={sourceName}
            src={sources[sourceName as 'light' | 'dark']}
            alt={alt}
            className={clsx(
              styles.themedImage,
              styles[`themedImage--${sourceName}`],
              className
            )}
            {...propsRest}
          />
        ) : null
      )}
```

Using `''` as `src` may lead to empty `img` tag on page. However, I haven't seen it in the inspector, so I may either not know where to look for it, or React somehow knows that when the `src` is empty, it shouldn't be rendered. Let me know what you think!